### ISSUE TITLE
MinutesForLongerTermTrend may be ignoring setting changes

### ISSUE THREAD

**Issue by GeoffBoyle** [mehtadone/PTFeeder#96]
I have changed my longer term trend to 48 hours with...
_"MinutesForLongerTermTrend": "2880"_ (previously set to 720). 
I restarted, no errors in console.

At the time of writing, the 720 minute change in price for ELFBTC was around 9%.
The 2,880 minute change in price for ELFBTC was around 16%.

The comment line in pairs.txt states 60 and 720, not 2880.
The comment also implies that the 720 minute value is being used.
I don't know whether this is cosmetic or if the bot is ignoring my changed value or does the bot need to run for 48 hours before using the longer trend?

Here's what was written to pairs.txt for ELFBTC...
```

#------------------BTC/ELF------------------
# Last Price: 0.00019477
# Volume: 8323.0594
# Price change (60): 2.91%
# Price change (720): 9%  <----------????
# Upside volatility: 0.88%
# Downside volatility: 0.6%

ELFBTC_trading_enabled = true
ELFBTC_max_cost_percentage = 1
ELFBTC_min_buy_balance = 0.1
ELFBTC_trailing_profit = 0.215
ELFBTC_trailing_buy = 0.215
ELFBTC_buy_value = -0.95
ELFBTC_buy_value_limit = 0
ELFBTC_DCA_enabled = -2
ELFBTC_sell_value = 1
ELFBTC_max_buy_spread = 0.75
ELFBTC_min_profit = 1
#------------------------------------
```

**Comment by mehtadone** [comment]
This is indeed a bug and thanks for reporting. 

The history on startup is not saving back far enough. The current version will start working after 2880 mins and this will be fixed in 1.3.5, hopefully end of this week. 

In terms of Price change (720), this should be showing 2880. Did you wait until PTF completed its run? Feels like there is a step missing somewhere.

**Comment by GeoffBoyle** [comment]
Not sure what I did now. 
After reporting the issue I reverted to 720 in case of problems, now I change it to 2880 and it does indeed show correctly in pairs.txt. 
That has to be down to me, not restarting properly or not waiting long enough after restart.
